Makefile

```diff
@@ -105,7 +105,7 @@ gosec:
 	./scripts/gosec.sh
 
 func-test:
-	go test -mod=vendor github.com/ceph/ceph-csi/e2e $(TESTOPTIONS)
+	go test -tags nautilus -mod=vendor github.com/ceph/ceph-csi/e2e $(TESTOPTIONS)
```

Comment:
instead of specifying tag in multiple places define a variable and use it

Reply:
@Madhu-1 sure, can that be a followup PR ?

Reply:
the change will be minimal as below, IMO as this PR is introducing this change we can do it here. 
```
ifndef CEPH_TAG
CEPH_TAG=nautilus
endif

go test -tags $(CEPH_TAG) -mod=vendor github.com/ceph/ceph-csi/e2e $(TESTOPTIONS)
```
the above tag need to be passed to other scripts also when its invoked from Makfile

Reply:
ok.. if that's the case, let me address in this PR itself. On it.

Reply:
@Madhu-1 ptal.. The changes required are in. For `travis-*.sh` its hardcoded. Let me know if any further changes are required.

Reply:
@Madhu-1 this has been addressed by building on top of #1182